onnxscript/optimizer/_constant_folding.py

```diff
@@ -168,7 +172,11 @@ def _get_numpy_value(val: ir.Value | None) -> np.ndarray | None:
         return None
     const_value = val.const_value
     if const_value is not None:
-        return const_value.numpy()
+        try:
+            return const_value.numpy()
+        except FileNotFoundError:
```

Comment:
Should this exception handling be in the implementation of the `numpy()` function? Wasn't sure how it might impact all other callers if we changed the signature of `numpy()` to return None sometimes. So, leaving it in its current form for now.

Reply:
numpy() should always return a numpy array or error. I think this is appropriate.